ethereum/contracts/zksync/interfaces/IMailbox.sol

```diff
@@ -118,13 +119,11 @@ interface IMailbox is IBase {
     ) external;
 
     function requestL2Transaction(
-        address _contractL2,
-        uint256 _l2Value,
+        L2Transaction memory _l2tx,
         bytes calldata _calldata,
-        uint256 _l2GasLimit,
-        uint256 _l2GasPerPubdataByteLimit,
         bytes[] calldata _factoryDeps,
-        address _refundRecipient
+        address _refundRecipient,
+        uint256 _baseAmount
     ) external payable returns (bytes32 canonicalTxHash);
```

Comment:
Keeping the function `payable` while it should not be when using the custom base token mode is not ideal but this way, we are avoiding breaking changes